package.json

```diff
@@ -45,7 +46,8 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
-    "@types/cytoscape": "^3.21.2",
+    "@types/cytoscape": "^3.21.4",
+    "@types/cytoscape-edgehandles": "^4.0.4",
```

Comment:
If there is a @types/cytoscape-edgehandles available, then you don't need a custom definition in index.d.ts.